code/modules/mob/living/living.dm

```diff
@@ -296,6 +296,23 @@
 
 /mob/living/proc/revive()
 	rejuvenate()
+	if(iscarbon(src))
```

Comment:
Is there a reason this is done here? Looks like it should be done in `/mob/living/carbon/revive()` after a `..()`